sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_v1/checks.sql

```diff
@@ -0,0 +1,2 @@
+#fail
+{{ is_within_row_count_for_historic_partitions(number_of_days=7, threshold_percentage=5) }}
```

Comment:
nit: 

Maybe we can think of a simpler name that is easier to remember? Maybe `past_partitions_within_threshold_percentage`? But not sure whether that is easier to remember or not. I'll leave it up to you to decide.